terraform/modules/hub/modules/ecs_fargate_app/security_groups.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_security_group" "lb" {
+  name        = "${local.identifier}-lb"
+  description = "${local.identifier}-lb"
+
+  vpc_id = var.vpc_id
+}
+
+output "lb_sg_id" {
+  value = aws_security_group.lb.id
+}
+
+resource "aws_security_group_rule" "task_ingress_from_lb" {
```

Comment:
can `microservice_connection` be used in place of these two security group rules?